### PR TITLE
developing shufflenet

### PR DESCRIPTION
**Pull request by doombeaker**
参考 https://github.com/pytorch/vision/blob/master/torchvision/models/shufflenetv2.py 实现 OneFlow 版本的 ShuffleNet

infer.py/infer.sh，train.py/train.sh 参考的 vgg 模型的相关脚本。

推理：
image

训练：
image

### CODE REVIEW

File: shufflenetv2/models/shufflenetv2.py [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+
+def shufflenetv2_x0dot5():
+    return ShuffleNetV2([4, 8, 4], [24, 48, 96, 192, 1024])
+
+
+def shufflenetv2_x1():
+    return ShuffleNetV2([4, 8, 4], [24, 116, 232, 464, 1024])
+
+
+if __name__ == "__main__":
```
Comment (Ldpe2G) [inline shufflenetv2/models/shufflenetv2.py:182]: 这段代码放 readme 里面吧，然后 main 函数删除掉

Comment (doombeaker) [reply shufflenetv2/models/shufflenetv2.py]: 已修改

**Review by Ldpe2G — APPROVED** [review]
